rb/spec/unit/selenium/webdriver/chrome/options_spec.rb

```diff
@@ -93,6 +93,35 @@ module Chrome
             subject.add_argument('foo')
             expect(subject.args).to include('foo')
           end
+
+          context 'multiple arguments' do
+            it 'should add only unique arguments' do
+              subject.add_argument('foo')
+              subject.add_argument('bar')
+
+              expect(subject.args).to eq %w[foo bar]
```

Comment:
If you're fixing up p0deje's request. can you wrap all your expectations in`()` instead of space separation and change line 105 to `it 'should not the add same argument more than once' do